srml/balances/src/lib.rs

```diff
@@ -464,6 +464,23 @@ decl_module! {
 			let dest = T::Lookup::lookup(dest)?;
 			<Self as Currency<_>>::transfer(&source, &dest, value)?;
 		}
+
+		/// Transfer some liquid free balance to another account, while checking that the transfer
+		/// will not kill the account.
+		///
+		/// 99% of the time you want `transfer` instead.
+		#[weight = SimpleDispatchInfo::FixedNormal(1_000_000)]
+		pub fn transfer_some(
```

Comment:
I feel that `_some` is not very meaningful suffix ?
maybe transfer_keep_alive.

Reply:
we could maybe merge both by adding an argument: ExistentialRequirement, but that would require a few more work on UI part I think

Reply:
I think we agreed above that not changing `transfer` and only adding an extra `transfer_x` function would work better. Renaming to `transfer_keep_alive` sounds good though.